lib/Page.js

```diff
@@ -81,6 +81,18 @@ Page.prototype.prepareTemplateData = function () {
   };
 };
 
+/**
+ * Records all h1-h3 headings id-text pair into this.headings
```

Comment:
Records all h1 to h3 headings ~~id-text pair~~ into this.headings

Mentioning id-text pair would reveal too much details imo.